evergreen/config.yml

```diff
@@ -1515,21 +1568,6 @@ buildvariants:
 #   tasks:
 #   - name: network_tests
 
-- name: rhel70
```

Comment:
Why are we removing this?

Reply:
We only had this builder to build artifacts for the baas server team and it's using a weird very old compiler.